.github/ISSUE_TEMPLATE/component-or-pattern.md

```diff
@@ -0,0 +1,70 @@
+---
+name: Component or pattern 💡
+about: Create a component or pattern for this project
+labels: 'status: needs triage'
+---
+
+### Summary
+
+> Summarise the component or pattern in one or two sentences.
+
+### Details
+
+> Link to design guidance.
+
+### Additional assets
+
+> Link to additional assets, for example, existing code, images, and
+> prototypes - provide as much useful information as you can.
+
+### Collaborators
+
+<!--
+  Add a name in each of the options below, for example:
+  - Design owner - John Smith
+-->
+
+- Design owner(s) -
+- Development peer reviewer(s) -
+- Sponsor user(s) -
+
+### Definition of done
+
+- [ ] Delivery follows the guidance in
+      [Contributing Guide](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/master/.github/CONTRIBUTING.md#5-make-a-pull-request)
+- [ ] Feedback from collaborators is addressed, and any follow-up issues are
+      created
+
+#### Testing
+
+- [ ] Tests are
+      [user-driven](https://testing-library.com/docs/guiding-principles) and
+      based on interactions defined in the design
+- [ ] Inputs and outputs are tested
+- [ ] [Accessibility tests](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/master/packages/experimental/scripts/generate/templates/DISPLAY_NAME-test.js#L16-L21)
+      for each use case
+- [ ] Targets 80% coverage
+
+#### Documentation
+
+- [ ] Example
+      [stories](https://storybook.js.org/docs/react/writing-stories/introduction)
+      capture and describe designed use cases
+- [ ] [Code examples for each use case, args tables, and docgen](https://storybook.js.org/docs/react/writing-docs/docs-page)
+      are available
+- [ ] Provides
+      [customization controls](https://storybook.js.org/docs/react/essentials/controls)
+      for interacting with inputs and modifiers
+- [ ] Additional usage documentation is accessible via the 'Docs' tab where
+      applicable
+
+#### Consistency
+
+- [ ] Props are consistent with Carbon-defined conventions, or otherwise
+      internally where unavailable
+- [ ] Custom class names can be supplied for additional style customization
+- [ ] Additional props are
```

Comment:
All props that have a functional/visual impact should be passed explicitly. What type of additional prop are we referring too?